### DISCUSSION TITLE
Cookies

### DISCUSSION THREAD

**Discussion by hayesti** [navidrome/navidrome#2848] [Ideas]
I use Navidrome on three separate devices using the same login. I notice that when moving between devices I am often logged out and need to reenter my login/password. It would be convenient to allow the user to remember their sessions to make the log in screen less frequent.

**Comment by metalheim** [comment]
You can, there is a configuration option `SessionTimeout` which Sets for how long the login sessions last. By default it is 24h

https://www.navidrome.org/docs/usage/configuration-options/#advanced-configuration

**Reply by hayesti** [reply]
I completely missed this. Thank you.
